tests/Util/JSONTest.php

```diff
@@ -100,49 +99,6 @@ public function encodeSubstitutesInvalidUtf8CharactersOnPhp72OrGreaterDataProvid
         ];
     }
 
-    /**
-     * @requires PHP < 7.2
-     *
-     * @dataProvider encodeSubstitutesInvalidUtf8CharactersOnPhp71OrLowerDataProvider
-     */
-    public function testEncodeSubstitutesInvalidUtf8CharactersOnPhp71OrLower($value, string $expectedResult): void
-    {
-        $this->assertSame($expectedResult, JSON::encode($value));
-    }
-
-    public function encodeSubstitutesInvalidUtf8CharactersOnPhp71OrLowerDataProvider(): \Generator
```

Comment:
Dropping this should mean that we could improve the `Json` class by using native escaping, right?

Reply:
You're right, I cleaned up the tests but not the code :pensive: